components/admin_console/billing/plan_details/plan_details.tsx

```diff
@@ -137,48 +145,50 @@ export const planDetailsTopElements = (userCount: number, isPaidTier: boolean, i
                 />
             </div>
         );
+
+        productName = (
+            <FormattedMessage
+                id='admin.billing.subscription.planDetails.productName.mmCloud'
+                defaultMessage='Mattermost Cloud'
+            />
+        );
     }
-    const productName = (
-        <FormattedMessage
-            id='admin.billing.subscription.planDetails.productName.cloudProfessional'
-            defaultMessage='Cloud Professional'
-        />
-    );
 
-    // switch (typeSubscription) {
-    // case 'FREE_TRIAL':
-    //     productName = (
-    //         <FormattedMessage
-    //             id='admin.billing.subscription.planDetails.productName.cloudProfessional'
-    //             defaultMessage='Cloud Professional'
-    //         />
-    //     );
-    //     break;
-    // case 'CLOUD_ENTERPRISE':
-    //     productName = (
-    //         <FormattedMessage
-    //             id='admin.billing.subscription.planDetails.productName.cloudEnterprise'
-    //             defaultMessage='Cloud Enterprise'
-    //         />
-    //     );
-    //     break;
-    // case 'CLOUD_STARTER':
-    //     productName = (
-    //         <FormattedMessage
-    //             id='admin.billing.subscription.planDetails.productName.cloudStarter'
-    //             defaultMessage='Cloud Starter'
-    //         />
-    //     );
-    //     break;
-    // default:
-    //     productName = (
-    //         <FormattedMessage
-    //             id='admin.billing.subscription.planDetails.productName.mmCloud'
-    //             defaultMessage='Mattermost Cloud'
-    //         />
-    //     );
-    //     break;
-    // }
+    switch (subscriptionPlan) {
+    case 'CLOUD_PROFESSIONAL':
+    case 'FREE_TRIAL':
```

Comment:
The free trial condition should not be part of the type of product check, as it could happen that the trial can be started for any plan in the future

Reply:
I believe this comment is still outstanding